app/controllers/proposals_controller.rb

```diff
@@ -99,7 +99,10 @@ def proposal_params
   end
 
   def require_speaker
-    raise ActiveRecord::RecordNotFound unless current_user.proposal_ids.include?(@proposal.id)
+    unless current_user.proposal_ids.include?(@proposal.id)
```

Comment:
Let me know if you'd prefer to leave this the way it was. Wasn't sure if there was a reason for having it that way